### ISSUE TITLE
TEOC sign in error

### ISSUE THREAD

**Issue by BCG-cmori** [OfficeDev/microsoft-teams-emergency-operations-center#165]
Hi seems like I might have a misconfiguration in my current configuration somewhere but not sure what exactly was misconfigured. 

![login error](https://github.com/OfficeDev/microsoft-teams-emergency-operations-center/assets/130778120/36e7b08a-4d5d-4f13-a766-a53fc97434e4)

Application manifest File

```
{
	"id": "5b56ed52-21c0-4ad3-9823-00c3db736d82",
	"acceptMappedClaims": null,
	"accessTokenAcceptedVersion": 2,
	"addIns": [],
	"allowPublicClient": null,
	"appId": "90cbf851-abf3-4c8a-8d42-8cecc45cbc5f",
	"appRoles": [],
	"oauth2AllowUrlPathMatching": false,
	"createdDateTime": "2024-01-23T14:25:08Z",
	"description": null,
	"certification": null,
	"disabledByMicrosoftStatus": null,
	"groupMembershipClaims": null,
	"identifierUris": [
		"api://teoc01p.azurewebsites.net/90cbf851-abf3-4c8a-8d42-8cecc45cbc5f"
	],
	"informationalUrls": {
		"termsOfService": null,
		"support": null,
		"privacy": null,
		"marketing": null
	},
	"keyCredentials": [],
	"knownClientApplications": [],
	"logoUrl": null,
	"logoutUrl": null,
	"name": "TEOC",
	"notes": null,
	"oauth2AllowIdTokenImplicitFlow": true,
	"oauth2AllowImplicitFlow": true,
	"oauth2Permissions": [
		{
			"adminConsentDescription": "Allows Teams to call the app’s web APIs as the current user.",
			"adminConsentDisplayName": "Teams can access app’s web APIs",
			"id": "2aa65e3f-e8f4-4e6f-960d-7345df0be882",
			"isEnabled": true,
			"lang": null,
			"origin": "Application",
			"type": "User",
			"userConsentDescription": "Allows Teams to call the app’s web APIs as the current user.",
			"userConsentDisplayName": "Teams can access app’s web APIs",
			"value": "access_as_user"
		}
	],
	"oauth2RequirePostResponse": false,
	"optionalClaims": {
		"idToken": [],
		"accessToken": [
			{
				"name": "idtyp",
				"source": null,
				"essential": false,
				"additionalProperties": []
			}
		],
		"saml2Token": []
	},
	"orgRestrictions": [],
	"parentalControlSettings": {
		"countriesBlockedForMinors": [],
		"legalAgeGroupRule": "Allow"
	},
	"passwordCredentials": [
		{
			"customKeyIdentifier": null,
			"endDate": "2025-01-22T14:28:02.121Z",
			"keyId": "dad2ef4e-97aa-4045-90a6-796be0309205",
			"startDate": "2024-01-23T14:28:02.121Z",
			"value": null,
			"createdOn": "2024-01-23T14:28:34.5487237Z",
			"hint": "dYx",
			"displayName": "TEOC Secret"
		}
	],
	"preAuthorizedApplications": [
		{
			"appId": "00000002-0000-0ff1-ce00-000000000000",
			"permissionIds": [
				"2aa65e3f-e8f4-4e6f-960d-7345df0be882"
			]
		},
		{
			"appId": "d3590ed6-52b3-4102-aeff-aad2292ab01c",
			"permissionIds": [
				"2aa65e3f-e8f4-4e6f-960d-7345df0be882"
			]
		},
		{
			"appId": "4765445b-32c6-49b0-83e6-1d93765276ca",
			"permissionIds": [
				"2aa65e3f-e8f4-4e6f-960d-7345df0be882"
			]
		},
		{
			"appId": "4345a7b9-9a63-4910-a426-35363201d503",
			"permissionIds": [
				"2aa65e3f-e8f4-4e6f-960d-7345df0be882"
			]
		},
		{
			"appId": "1fec8e78-bce4-4aaf-ab1b-5451cc387264",
			"permissionIds": [
				"2aa65e3f-e8f4-4e6f-960d-7345df0be882"
			]
		},
		{
			"appId": "5e3ce6c0-2b1f-4285-8d4b-75ee78787346",
			"permissionIds": [
				"2aa65e3f-e8f4-4e6f-960d-7345df0be882"
			]
		}
	],
	"publisherDomain": "baltimorecountymd.gov",
	"replyUrlsWithType": [
		{
			"url": "https://teoc01p.azurewebsites.net/blank-auth-end.html",
			"type": "Spa"
		},
		{
			"url": "https://teoc01p.azurewebsites.net/auth-end.html?clientId=90cbf851-abf3-4c8a-8d42-8cecc45cbc5f",
			"type": "Spa"
		},
		{
			"url": "https://teoc01p.azurewebsites.net/auth-end.html",
			"type": "Web"
		}
	],
	"requiredResourceAccess": [
		{
			"resourceAppId": "00000003-0000-0000-c000-000000000000",
			"resourceAccess": [
				{
					"id": "0e263e50-5827-48a4-b97c-d940288653c7",
					"type": "Scope"
				},
				{
					"id": "5f8c59db-677d-491f-a6b8-5f174b11ec1d",
					"type": "Scope"
				},
				{
					"id": "4e46008b-f24c-477d-8fff-7bb4ec7aafe0",
					"type": "Scope"
				},
				{
					"id": "e383f46e-2787-4529-855e-0e479a3ffac0",
					"type": "Scope"
				},
				{
					"id": "a65f2972-a4f8-4f5e-afd7-69ccb046d5dc",
					"type": "Scope"
				},
				{
					"id": "ba47897c-39ec-4d83-8086-ee8256fa737d",
					"type": "Scope"
				},
				{
					"id": "65e50fdc-43b7-4915-933e-e8138f11f40a",
					"type": "Scope"
				},
				{
					"id": "f45671fb-e0fe-4b4b-be20-3d3ce43f1bcb",
					"type": "Scope"
				},
				{
					"id": "2219042f-cab5-40cc-b0d2-16b1540b4c5f",
					"type": "Scope"
				},
				{
					"id": "4a06efd2-f825-4e34-813e-82a57b03d1ee",
					"type": "Scope"
				},
				{
					"id": "0f4595f7-64b1-4e13-81bc-11a249df07a9",
					"type": "Scope"
				},
				{
					"id": "a9ff19c2-f369-4a95-9a25-ba9d460efc8e",
					"type": "Scope"
				},
				{
					"id": "539dabd7-b5b6-4117-b164-d60cd15a8671",
					"type": "Scope"
				},
				{
					"id": "e1fe6dd8-ba31-4d61-89e7-88639da4683d",
					"type": "Scope"
				},
				{
					"id": "b340eb25-3456-403f-be2f-af7a0d370277",
					"type": "Scope"
				}
			]
		}
	],
	"samlMetadataUrl": null,
	"signInUrl": null,
	"signInAudience": "AzureADMyOrg",
	"tags": [],
	"tokenEncryptionKeyId": null
}
```

**Comment by sanjaymittaluk** [comment]
> Hi seems like I might have a misconfiguration in my current configuration somewhere but not sure what exactly was misconfigured.
> 
> ![login error](https://private-user-images.githubusercontent.com/130778120/299705599-36e7b08a-4d5d-4f13-a766-a53fc97434e4.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDYxOTgwNTAsIm5iZiI6MTcwNjE5Nzc1MCwicGF0aCI6Ii8xMzA3NzgxMjAvMjk5NzA1NTk5LTM2ZTdiMDhhLTRkNWQtNGYxMy1hNzY2LWE1M2ZjOTc0MzRlNC5QTkc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEyNVQxNTQ5MTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xZTVmZjEwNWQ2NzA1OGQ1Yzk4NDI2Y2UwYWYyM2YwNTM4NTFjNjEwOGZiZDIyN2FjY2E4MDA0YzhkOTk1YjlhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.SpoJXXj2cVCzbEw5ywXPUJwZbUEQPz8YGd1IMO-wzLw)
> 
> Application manifest File
> 
> ```
> {
> 	"id": "5b56ed52-21c0-4ad3-9823-00c3db736d82",
> 	"acceptMappedClaims": null,
> 	"accessTokenAcceptedVersion": 2,
> 	"addIns": [],
> 	"allowPublicClient": null,
> 	"appId": "90cbf851-abf3-4c8a-8d42-8cecc45cbc5f",
> 	"appRoles": [],
> 	"oauth2AllowUrlPathMatching": false,
> 	"createdDateTime": "2024-01-23T14:25:08Z",
> 	"description": null,
> 	"certification": null,
> 	"disabledByMicrosoftStatus": null,
> 	"groupMembershipClaims": null,
> 	"identifierUris": [
> 		"api://teoc01p.azurewebsites.net/90cbf851-abf3-4c8a-8d42-8cecc45cbc5f"
> 	],
> 	"informationalUrls": {
> 		"termsOfService": null,
> 		"support": null,
> 		"privacy": null,
> 		"marketing": null
> 	},
> 	"keyCredentials": [],
> 	"knownClientApplications": [],
> 	"logoUrl": null,
> 	"logoutUrl": null,
> 	"name": "TEOC",
> 	"notes": null,
> 	"oauth2AllowIdTokenImplicitFlow": true,
> 	"oauth2AllowImplicitFlow": true,
> 	"oauth2Permissions": [
> 		{
> 			"adminConsentDescription": "Allows Teams to call the app’s web APIs as the current user.",
> 			"adminConsentDisplayName": "Teams can access app’s web APIs",
> 			"id": "2aa65e3f-e8f4-4e6f-960d-7345df0be882",
> 			"isEnabled": true,
> 			"lang": null,
> 			"origin": "Application",
> 			"type": "User",
> 			"userConsentDescription": "Allows Teams to call the app’s web APIs as the current user.",
> 			"userConsentDisplayName": "Teams can access app’s web APIs",
> 			"value": "access_as_user"
> 		}
> 	],
> 	"oauth2RequirePostResponse": false,
> 	"optionalClaims": {
> 		"idToken": [],
> 		"accessToken": [
> 			{
> 				"name": "idtyp",
> 				"source": null,
> 				"essential": false,
> 				"additionalProperties": []
> 			}
> 		],
> 		"saml2Token": []
> 	},
> 	"orgRestrictions": [],
> 	"parentalControlSettings": {
> 		"countriesBlockedForMinors": [],
> 		"legalAgeGroupRule": "Allow"
> 	},
> 	"passwordCredentials": [
> 		{
> 			"customKeyIdentifier": null,
> 			"endDate": "2025-01-22T14:28:02.121Z",
> 			"keyId": "dad2ef4e-97aa-4045-90a6-796be0309205",
> 			"startDate": "2024-01-23T14:28:02.121Z",
> 			"value": null,
> 			"createdOn": "2024-01-23T14:28:34.5487237Z",
> 			"hint": "dYx",
> 			"displayName": "TEOC Secret"
> 		}
> 	],
> 	"preAuthorizedApplications": [
> 		{
> 			"appId": "00000002-0000-0ff1-ce00-000000000000",
> 			"permissionIds": [
> 				"2aa65e3f-e8f4-4e6f-960d-7345df0be882"
> 			]
> 		},
> 		{
> 			"appId": "d3590ed6-52b3-4102-aeff-aad2292ab01c",
> 			"permissionIds": [
> 				"2aa65e3f-e8f4-4e6f-960d-7345df0be882"
> 			]
> 		},
> 		{
> 			"appId": "4765445b-32c6-49b0-83e6-1d93765276ca",
> 			"permissionIds": [
> 				"2aa65e3f-e8f4-4e6f-960d-7345df0be882"
> 			]
> 		},
> 		{
> 			"appId": "4345a7b9-9a63-4910-a426-35363201d503",
> 			"permissionIds": [
> 				"2aa65e3f-e8f4-4e6f-960d-7345df0be882"
> 			]
> 		},
> 		{
> 			"appId": "1fec8e78-bce4-4aaf-ab1b-5451cc387264",
> 			"permissionIds": [
> 				"2aa65e3f-e8f4-4e6f-960d-7345df0be882"
> 			]
> 		},
> 		{
> 			"appId": "5e3ce6c0-2b1f-4285-8d4b-75ee78787346",
> 			"permissionIds": [
> 				"2aa65e3f-e8f4-4e6f-960d-7345df0be882"
> 			]
> 		}
> 	],
> 	"publisherDomain": "baltimorecountymd.gov",
> 	"replyUrlsWithType": [
> 		{
> 			"url": "https://teoc01p.azurewebsites.net/blank-auth-end.html",
> 			"type": "Spa"
> 		},
> 		{
> 			"url": "https://teoc01p.azurewebsites.net/auth-end.html?clientId=90cbf851-abf3-4c8a-8d42-8cecc45cbc5f",
> 			"type": "Spa"
> 		},
> 		{
> 			"url": "https://teoc01p.azurewebsites.net/auth-end.html",
> 			"type": "Web"
> 		}
> 	],
> 	"requiredResourceAccess": [
> 		{
> 			"resourceAppId": "00000003-0000-0000-c000-000000000000",
> 			"resourceAccess": [
> 				{
> 					"id": "0e263e50-5827-48a4-b97c-d940288653c7",
> 					"type": "Scope"
> 				},
> 				{
> 					"id": "5f8c59db-677d-491f-a6b8-5f174b11ec1d",
> 					"type": "Scope"
> 				},
> 				{
> 					"id": "4e46008b-f24c-477d-8fff-7bb4ec7aafe0",
> 					"type": "Scope"
> 				},
> 				{
> 					"id": "e383f46e-2787-4529-855e-0e479a3ffac0",
> 					"type": "Scope"
> 				},
> 				{
> 					"id": "a65f2972-a4f8-4f5e-afd7-69ccb046d5dc",
> 					"type": "Scope"
> 				},
> 				{
> 					"id": "ba47897c-39ec-4d83-8086-ee8256fa737d",
> 					"type": "Scope"
> 				},
> 				{
> 					"id": "65e50fdc-43b7-4915-933e-e8138f11f40a",
> 					"type": "Scope"
> 				},
> 				{
> 					"id": "f45671fb-e0fe-4b4b-be20-3d3ce43f1bcb",
> 					"type": "Scope"
> 				},
> 				{
> 					"id": "2219042f-cab5-40cc-b0d2-16b1540b4c5f",
> 					"type": "Scope"
> 				},
> 				{
> 					"id": "4a06efd2-f825-4e34-813e-82a57b03d1ee",
> 					"type": "Scope"
> 				},
> 				{
> 					"id": "0f4595f7-64b1-4e13-81bc-11a249df07a9",
> 					"type": "Scope"
> 				},
> 				{
> 					"id": "a9ff19c2-f369-4a95-9a25-ba9d460efc8e",
> 					"type": "Scope"
> 				},
> 				{
> 					"id": "539dabd7-b5b6-4117-b164-d60cd15a8671",
> 					"type": "Scope"
> 				},
> 				{
> 					"id": "e1fe6dd8-ba31-4d61-89e7-88639da4683d",
> 					"type": "Scope"
> 				},
> 				{
> 					"id": "b340eb25-3456-403f-be2f-af7a0d370277",
> 					"type": "Scope"
> 				}
> 			]
> 		}
> 	],
> 	"samlMetadataUrl": null,
> 	"signInUrl": null,
> 	"signInAudience": "AzureADMyOrg",
> 	"tags": [],
> 	"tokenEncryptionKeyId": null
> }
> ```

I think you need to check that the Manifest code is updated and correct as per the documentation

**Comment by BCG-cmori** [comment]
I have and I am not seeing anything that's misconfigured. Where should I check to see if it is misconfigured?

**Comment by BCG-cmori** [comment]
Issue literally just resolved itself. However I am having issues submitting the incidents :( 

![new error](https://github.com/OfficeDev/microsoft-teams-emergency-operations-center/assets/130778120/8ed17978-51dd-40c8-91b0-6964d2730742)

**Comment by BCG-cmori** [comment]
Here are some screenshots of the groups policy. However I do not want to remove the Naming Policy. 

![Group Settings](https://github.com/OfficeDev/microsoft-teams-emergency-operations-center/assets/130778120/159b7e59-f101-4360-9dcb-a0034459014d)

**Comment by sanjaymittaluk** [comment]
> Issue literally just resolved itself. However I am having issues submitting the incidents :(
> 
> ![new error](https://private-user-images.githubusercontent.com/130778120/299739698-8ed17978-51dd-40c8-91b0-6964d2730742.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDYyMDE3MjQsIm5iZiI6MTcwNjIwMTQyNCwicGF0aCI6Ii8xMzA3NzgxMjAvMjk5NzM5Njk4LThlZDE3OTc4LTUxZGQtNDBjOC05MWIwLTY5NjRkMjczMDc0Mi5QTkc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEyNVQxNjUwMjRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zYWMwNjA1YjU0M2VjOTAzODMyYjZiMTMyMDQwZTljODM2MDIzNGI2NDE3ZTY1NzE1MmFjMDlmNGI1ZGU1MTg4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.-zJVaKUjqgoGI8aCNMlb50ddtCfC_keZEm-QNA594Og)

I am waiting for a response for the very same problem. I think there is an issue; Perhaps you can attach the console log and tracers for the admins to look and then you can take it from there.

**Comment by v-saikirang** [comment]
Hi @BCG-cmori , @sanjaymittaluk 

There is a known issue with one of the graph end points causing the incident creation to fail. We are working with the team and this is expected to be resolved ASAP. We will keep you posted. Thanks for your patience on this.

Thanks.

**Comment by sanjaymittaluk** [comment]
> There is a known issue with one of the graph end points, causing the incident creation to fail. We are working with the team and this is expected to be resolved ASAP. We will keep you posted. Thanks for your patience on this.

thank you. much appreciated.

**Comment by v-saikirang** [comment]
Hi @BCG-cmori,

It is expected to be resolved by Jan 31st. We will keep you posted.

Thanks.

**Comment by BCG-cmori** [comment]
Great thanks!

**Comment by BCG-cmori** [comment]
Hi just wanted to follow up on this to see if we have any good news?

**Comment by v-asshrirao** [comment]
@BCG-cmori,
Thank you for your patience on this.
We have released the hotfix for this issue. Request you to kindly sync the code by following steps mentioned in this [link](https://github.com/OfficeDev/microsoft-teams-emergency-operations-center/wiki/Upgrade-to-Latest-Version#3-sync-latest-version)

If you are still getting the issue we suggest you to clear cache and try again.

**Comment by BCG-cmori** [comment]
Awesome thank you! I will work on this and follow up if we run into any issues.

**Comment by v-asshrirao** [comment]
@BCG-cmori, We will close this ticket now. In case of any issue, you can reopen this or create new ticket. Thank you.

**Comment by BCG-cmori** [comment]
Hi

I did the sync and cleared cache in my browser but doesn't seem to be working yet. Do I need do anything else besides those items?

**Comment by v-asshrirao** [comment]
@BCG-cmori, Can you please confirm if sync was successful. Please share the screenshot of deployment logs from App Service > Deployment Center > Logs
If sync is successful, can you please try to open teams in different browser, may be the one which you have not used or in Teams client.
If you are still facing the issue, request you to kindly share the console logs and network trace.

**Comment by BCG-cmori** [comment]
Yeah i have waited 24 hours now and tried different browsers and I am still getting an error when i try to submit a new incident. I also looked at the developer console logs and looks like its specific to the prefix we force on new groups. Do we need to disable that or is there a way for TEOC to follow the prefix we have set? 


![teoc-screenshot-sync](https://github.com/OfficeDev/microsoft-teams-emergency-operations-center/assets/130778120/3f5614bb-8270-4a20-b788-b199a74eabca)
![teoc-screenshot-sync-2](https://github.com/OfficeDev/microsoft-teams-emergency-operations-center/assets/130778120/339ce6cb-3b34-4903-975c-bee626a4ee08)
